integration-templates/ashby/ashby-jobs.ts

```diff
@@ -41,8 +41,8 @@ async function saveAllJobs(nango: NangoSync, jobslastsyncToken: string) {
         const metadata = (await nango.getMetadata()) || {};
         metadata['jobslastsyncToken'] = jobslastsyncToken;
         await nango.setMetadata(metadata);
-    } catch (error) {
-        console.error('Error occurred while fetching and saving jobs:', error);
+    } catch (error: any) {
```

Comment:
Error is unknown not any
```suggestion
    } catch (error) {
```

Reply:
@bodinsamuel, your suggestion is appreciated. It seems like we need to update all template syncs to include a catch block like this: `} catch (error) { ... ${error} }`. Could you please confirm this adjustment so that I can proceed with updating all the necessary files? Thank you